### PR TITLE
Fix when file variable is array, or multer option putSingleFilesInArray is true

### PR DESCRIPTION
**Pull request by rferro**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -22,17 +22,22 @@ module.exports = function(req, res, next) {
 			for(var key in req.files) {
 				if (req.files.hasOwnProperty(key)) {
 					file = req.files[key];
-					if (err && (options && !options.reapOnError)) {
-						debug('skipped auto removal of %s - please manually deprecate.',  file.path);
-						continue;
+					if (!Array.isArray(file)) {
```
Comment (JcBernack) [inline index.js]: I would suggest to use `!(file instanceof Array)` instead for compatibility reasons.

**Comment by rferro** [comment]
Tks JcBernack

**Comment by joelabair** [comment]
thx @rferro !

**Comment by rferro** [comment]
thx @joelabair

**Pull request merged**